app/javascript/react/screens/App/Overview/screens/MappingWizard/MappingWizard.scss

```diff
@@ -62,3 +62,16 @@
   border-color: #39a5dc;
   color: #fff;
 }
+
+.inner-modal-backdrop {
+  z-index: 1060; // .modal-dialog defaults to z-index: 1050
+}
+
+.inner-modal-dialog {
+  z-index: 1070;
+  margin-top: 150px;
+}
+
+.confirm-warning-icon {
+  font-size: 3em;
+}
```

Comment:
Is it weird that this css lives in `MappingWizard.scss`?

Reply:
oh oops. I meant to put that all in PlanWizard... and it should be moved somewhere more common anyway. Good catch.